testing/tests/DevExpress.ui.widgets/popup.tests.js

```diff
@@ -196,27 +196,11 @@ QUnit.test("done button is located after cancel button in non-win8 device", func
     instance.option("toolbarItems", [{ shortcut: "cancel" }, { shortcut: "done" }]);
     $popupBottom = instance.$content().parent().find(".dx-popup-bottom");
     assert.equal($popupBottom.text(), "CancelOK", "buttons order is correct");
-});
-
-QUnit.test("done button is located before cancel button in win10", function(assert) {
-    devices.current("win10");
-
-    var $popup = $("#popup").dxPopup({
-            toolbarItems: [{ shortcut: "cancel" }, { shortcut: "done" }],
-            animation: null,
-            visible: true
-        }),
-        instance = $popup.dxPopup("instance"),
-        $popupBottom = instance.$content().parent().find(".dx-popup-bottom");
-
-    assert.equal($popupBottom.text(), "OKCancel", "buttons order is correct");
-
-    instance.option("toolbarItems", [{ shortcut: "done" }, { shortcut: "cancel" }]);
-    $popupBottom = instance.$content().parent().find(".dx-popup-bottom");
-    assert.equal($popupBottom.text(), "OKCancel", "buttons order is correct");
+    devices.current(devices.real());
 });
 
 QUnit.test("buttons should be rendered correctly after toolbar was repainted", function(assert) {
+    devices.current("desktop");
```

Comment:
Why didn't we need this before?

Reply:
In the old version, we use current devices "win" set in the previous test. I have fixed tests to use actual devices, but these tests failed for some devices because some devices have another buttons order by default.